ffmuc-eol-ssid/luasrc/lib/gluon/upgrade/950-ffmuc-update-nagger

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/lua
+
+local util = require 'gluon.util'
+local uci = require('simple-uci').cursor()
+
+-- Reuse IBSS mac address
+local macaddr = util.generate_mac(2)
+if not macaddr then
+	return
+end
+
+uci:section('wireless', 'wifi-iface', 'dep_radio0', {
+	ifname = 'eolnotice0',
+	network = 'client',
+	disabled = '0',
+	device = 'radio0',
+	mode = 'ap',
+	macaddr = macaddr,
+	ssid = 'supportende.muenchen.freifunk.net',
```

Comment:
This does not work as the max length of SSID strings is 32 characters (while this string are 33)
So I would suggest something with "ffmuc.net" as it is shorter than "muenchen.ffmuc.net"